custodian/policies/s3/s3-check-missing-ssl.yml

```diff
@@ -0,0 +1,48 @@
+policies:
+  - name: s3-check-missing-ssl-policy
+    mode:
+      type: periodic
+      schedule: "rate(1 hour)"
+      role: arn:aws:iam::{account_id}:role/CustodianCheckMissingSSLPolicyS3
+    resource: aws.s3
+    description: |
+      Scans all S3 buckets and marks buckets missing an AllowSSLRequestsOnly for deletion
```

Comment:
Is that description correct. Thought the script adds the missing policy?

Reply:
No, I originally did it with the tag and delete method before I realised you could add the policy in. I've changed the description.

Reply:
I've actually pushed the change to the description now.